docs/development/local_development.md

```diff
@@ -0,0 +1,21 @@
+# Running on development mode
+
+The following document highlights how to deploy a Build operator locally for running on development mode.
```

Comment:
```suggestion
The following document highlights how to deploy the Shipwright operator locally for development.
```

Reply:
@adambkaplan on this one I think is `Build`. The operator is and have always been the `Build`, which defines the different sets of controllers(Build, BuildRun, ClusterBuildStrategy and BuildStrategy). Shipwright is just the org name, not the project neither operator name.

Reply:
In Travis we are publishing the image as `quay.io/shipwright/shipwright-operator` - we should be consistent.